src/inmanta/app.py

```diff
@@ -83,6 +83,7 @@
 
 @command("server", help_msg="Start the inmanta server")
 def start_server(options: argparse.Namespace) -> None:
+    LOGGER.info(f"Starting inmanta-server version {COMPILER_VERSION}")
```

Comment:
Make sure to use the `%s` notation.
```suggestion
    LOGGER.info(f"Starting inmanta-server version %s", COMPILER_VERSION)
```